wgsl/index.bs

```diff
@@ -5709,21 +5776,30 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
     | [=syntax/additive_expression=] [=syntax/minus=] [=syntax/multiplicative_expression=]
 </div>
 <div class='syntax' noexport='true'>
-  <dfn for=syntax>relational_expression</dfn> :
+  <dfn for=syntax>shift_expression</dfn> :
 
     | [=syntax/additive_expression=]
 
-    | [=syntax/additive_expression=] [=syntax/less_than=] [=syntax/additive_expression=]
+    | [=syntax/unary_expression=] [=syntax/shift_left=] [=syntax/unary_expression=]
+
+    | [=syntax/unary_expression=] [=syntax/shift_right=] [=syntax/unary_expression=]
+</div>
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>relational_expression</dfn> :
+
+    | [=syntax/shift_expression=]
 
-    | [=syntax/additive_expression=] [=syntax/greater_than=] [=syntax/additive_expression=]
+    | [=syntax/shift_expression=] [=syntax/less_than=] [=syntax/shift_expression=]
```

Comment:
Why did the ordering of `>`, `<=`, etc change?

Reply:
Artifact.  I started with a plain revert of the previous removal.